packages/ember/tests/routing/substates_test.js

```diff
@@ -415,6 +415,41 @@ QUnit.test('Default error event moves into nested route', function() {
   equal(appController.get('currentPath'), 'grandma.error', 'Initial route fully loaded');
 });
 
+QUnit.test('Error events that aren\'t bubbled don\t throw application assertions', function() {
+  expect(2);
```

Comment:
ya, I understood that it failed on 2.1.  I just didn't understand where the second assertion was coming from, but you clarified that above